### PR TITLE
Short summery of Vinnova AI convent

### PR DESCRIPTION
**Pull request by strobba**

### CODE REVIEW

File: content/posts/ai_konferens_2024-02-07.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+title: Summering av eventet AI i klimatets tjänst onsdagen 7 februari 2024
+date: 2024-02-08
+modified:
+author: Christoffer Strömberg
+category: Event
```
Comment (mgcth) [inline content/posts/ai_konferens_2024-02-07.md:5]: Should it be under event or _Projekt_, as the other course I attended is labelled as _Projekt_.

File: content/posts/ai_konferens_2024-02-07.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+Programkonferens AI i klimatets tjänst, 7 februari 2024 - YouTube</a>).
+Eftermiddagen var öppen för allmän beskådan på youtube. Jag lyssnade in
+på konferensen och ska här kort summera eftermiddagen.
+
+Eftermiddagen börjar med tre korta projektpresentationer:
+
+* Projekt för att skapa större trovärdighet i klimatberäkningar, bla. används
+AI för att utvärdera CO2 läckage mellan länder (se
+<a href="https://2050.se/en/aica-ai-powered-carbon-border-adjustments-eng"
+target="_blank">AICA at 2050.se</a>).
```
Comment (mgcth) [inline content/posts/ai_konferens_2024-02-07.md:21]: This link text looks strange.

File: content/posts/ai_konferens_2024-02-07.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+* Digitala tvillingar för byggnader i syfte att effektivisera renovering
+(se <a href="https://chalmersindustriteknik.se/projekt/decarbonaite/"
+target = "_blank">DecarbonAIte</a>).
+* Optimering av algnäring för att minimera metangasproduktion hos kor.
+
+Gemensamt för alla projekt var problem att hitta kvalitativ data.
+
+Näst upp var Mark Bünger, som är en av människorna bakom
+<a href="https://www.plantiver.se/" target="_blank">plantiverse</a>, där dom via
+AI gett växter mänskliga verktyg, så som pengar. Detta så att växter ska kunna
```
Comment (mgcth) [inline content/posts/ai_konferens_2024-02-07.md:31]: Känns som rätt väg framåt.

File: content/posts/ai_konferens_2024-02-07.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+att rädda klimatet.
+* Håll det enkelt och skapa platformar för data.
+* Satsa på lågt hängande frukter.
+
+Vidare gick David igenom hur en bör tänka när en väljer AI projekt, bla bör man
+akta sig för all buzzwords som tex AI.
+
+Slutligen bör jag nämna att jag spelade en hel del skitgubbe med min son som var
+krasslig och hemma från föskolan denna dag, så jag kan har missat några
+fantastiska detaljer.
```
Comment (mgcth) [inline content/posts/ai_konferens_2024-02-07.md:60]: ❤️ but should perhaps be redacted? 😁

**Review by mgcth — COMMENTED** [review]
Very nice summary @strobba, sad that it seemed so desperately out of touch.

File: content/posts/ai_konferens_2024-02-07.md (hunk 1 above)
Comment (strobba) [reply content/posts/ai_konferens_2024-02-07.md:5]: updated to Projekt.

File: content/posts/ai_konferens_2024-02-07.md (hunk 2 above)
Comment (strobba) [reply content/posts/ai_konferens_2024-02-07.md:21]: True, updated.

File: content/posts/ai_konferens_2024-02-07.md (hunk 4 above)
Comment (strobba) [reply content/posts/ai_konferens_2024-02-07.md:60]: removed.

**Comment by docNord** [comment]
Can you change somewhere in the text to acknowledge that you are the author (otherwise it seems I have written it since my name is at the bottom as publisher), which would be fine except it is written in "I"-form. :)

**Comment by mgcth** [comment]
> Can you change somewhere in the text to acknowledge that you are the author (otherwise it seems I have written it since my name is at the bottom as publisher), which would be fine except it is written in "I"-form. :)

Perhaps we should fix that in the template, as we provide the author in the metadata. We could imply write it out.

**Comment by strobba** [comment]
I removed "jag-perspektivet".

File: content/posts/ai_konferens_2024-02-07.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
+title: Summering av eventet AI i klimatets tjänst onsdagen 7 februari 2024
+date: 2024-02-08
```
Comment (docNord) [inline content/posts/ai_konferens_2024-02-07.md:2]: 2024-02-07 to make it match the date of the title.

**Review by docNord — APPROVED** [review]
Please fix the date such that file name and metadata matches, other than that great work!

File: content/posts/ai_konferens_2024-02-07.md (hunk 5 above)
Comment (strobba) [reply content/posts/ai_konferens_2024-02-07.md]: done

**Pull request merged**